integration/bundle/helpers_test.go

```diff
@@ -119,6 +119,15 @@ func runResource(t testutil.TestingT, ctx context.Context, path, key string) (st
 	return stdout.String(), err
 }
 
+func runResourceWithStderr(t testutil.TestingT, ctx context.Context, path, key string) (string, string, error) {
```

Comment:
In this and other test helpers, how about not returning the error but instead reporting the error via t.Error or t.Fatal? This makes test case code cleaner.